readme.md

```diff
@@ -64,6 +64,17 @@ Prettify the file paths in the stack:
 
 `/Users/sindresorhus/dev/clean-stack/unicorn.js:2:15` → `~/dev/clean-stack/unicorn.js:2:15`
 
+##### basePath
+
+Type: `string`<br>
+Default: `undefined`
+
+Remove all occurences of `basePath`. Useful for removing absolute paths.
```

Comment:
I think this be clearer about it only removing the start of stack trace paths.

Reply:
I guess we can match for the starting `(`, but I'm not totally sure it's always there.

Reply:
I guess we can match `/(at | \()/` to catch both ` (/path` and `at /path`.

Reply:
Fixed in https://github.com/sindresorhus/clean-stack/pull/18/commits/8386b39f24e9558d5ec1a470f8a98a4411a5752a.